[Bug]: missing @babel/core in remix-eslint-config

### Which Remix packages are impacted?

- [ ] `remix` (Remix core)
- [ ] `create-remix`
- [ ] `@remix-run/architect`
- [ ] `@remix-run/cloudflare-workers`
- [ ] `@remix-run/dev`
- [ ] `@remix-run/express`
- [ ] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [ ] `@remix-run/react`
- [ ] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

1.1.0

### What version of Node are you using? Minimum supported version is 14.

14.16.0

### Steps to Reproduce

1. access https://stackblitz.com/edit/github-1ywvff?file=package.json
2. run `npm run eslint`

### Expected Behavior

Eslint will run correctly

### Actual Behavior

```
Error: Failed to load parser '@babel/eslint-parser' declared in '.eslintrc » @remix-run/eslint-config': Cannot find module '@babel/core/package.json'
```

--- a/packages/remix-eslint-config/package.json
+++ b/packages/remix-eslint-config/package.json
@@ -10,6 +10,7 @@
     "jest.js"
   ],
   "dependencies": {
+    "@babel/core": "^7.17.2",
     "@babel/eslint-parser": "^7.16.3",
     "@babel/preset-react": "^7.16.0",
     "@rushstack/eslint-patch": "^1.1.0",

